clients/src/test/java/org/apache/kafka/clients/consumer/internals/LegacyKafkaConsumerTest.java

```diff
@@ -2582,7 +2581,7 @@ private FetchResponse fetchResponse(TopicPartition partition, long fetchOffset,
         return fetchResponse(Collections.singletonMap(partition, fetchInfo));
     }
 
-    private KafkaConsumer<String, String> newConsumer(Time time,
+    private LegacyKafkaConsumer<String, String> newConsumer(Time time,
                                                       KafkaClient client,
```

Comment:
nit: The indentation of the arguments are not incorrect in a few methods.

Reply:
Thanks. Fixed.